articles/multifactor-authentication/administrator/sms-templates.md

```diff
@@ -0,0 +1,20 @@
+---
+description: Customize SMS Messages
+---
+
+# Customize SMS Messages
+
+If you need to change the content of the messages that are sent by Auth0 when you enroll (when you associate a phone number with guardian) or verify (when you proof that have that device).
+
+First, go to ([Multifactor Auth With Guardian](${manage_url}/#/guardian), then click on the **SMS** box to configure your SMS settings.
+
+![](/media/articles/mfa/sms-config.png)
+
+You have two fields to customize your messages:
+* Enrollment Template: this message is sent by Auth0 to associate a user with a phone number. 
+* Verification Template: this message is sent by Auth0 to verify the possesion of the phone. 
```

Comment:
Verification Template: the message sent by Auth0 to verify the possession of the device.